tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot

```diff
@@ -410,32 +411,12 @@ Send Email
     \    ${remarks}=  Get From Dictionary  ${PLUGIN_TEST_RESULTS_DICT}  ${key}
     \    Append To File  email_body.txt  ${remarks}${\n}
 
-    ${installer_notes_str}=  Run  cat ${TEST_SCRIPTS_ROOT}/18-1-VIC-UI-Installer.robot | grep "# NOTE" | sed -e "s/^[[:space:]]*//g"
-    ${uninstaller_notes_str}=  Run  cat ${TEST_SCRIPTS_ROOT}/18-2-VIC-UI-Uninstaller.robot | grep "# NOTE" | sed -e "s/^[[:space:]]*//g"
-    ${upgrader_notes_str}=  Run  cat ${TEST_SCRIPTS_ROOT}/18-3-VIC-UI-Upgrader.robot | grep "# NOTE" | sed -e "s/^[[:space:]]*//g"
+    ${flex_note}=  Catenate
+    ...  Due to challenges in getting the right environment for the automated test framework to run and the fact that
+    ...  the amount of time and effort spent on automating the test for the obsolete Flex Client plugin is unreasonably big,
+    ...  we decided to run Flex Client plugin tests manually before each major milestone (e.g. release).
```

Comment:
I would modify/shorten a bit to something like "Due to challenges surrounding Flex testing automation, Flex Client plugin tests are manually run after any Flex code change and before each major milestone (e.g. release)."